src/Http/Authentication.Abstractions/src/AuthenticationProperties.cs

```diff
@@ -101,6 +102,16 @@ public string RedirectUri
             set => SetBool(RefreshKey, value);
         }
 
+
+        /// <summary>
+        /// Gets or sets if claims have been transformed.
+        /// </summary>
+        public bool? ClaimsTransformed
```

Comment:
Does this need a property if we don't expect anyone else to use it?

Reply:
Well this is helpful for us, are you saying just make it internal/move the helper to an internal extension method in the implementation(Core) assembly instead?